stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -63,38 +63,26 @@ export
     trsm
 
 
-const libblas = Base.libblas_name
-const liblapack = Base.liblapack_name
+const libblas = "libblastrampoline"
+const liblapack = "libblastrampoline"
 
 import LinearAlgebra
 import LinearAlgebra: BlasReal, BlasComplex, BlasFloat, BlasInt, DimensionMismatch, checksquare, stride1, chkstride1, axpy!
 
 import Libdl
 
 # utility routines
-let lib = C_NULL
-global function determine_vendor()
-    if lib == C_NULL
-        lib = something(Libdl.dlopen(libblas; throw_error=false), C_NULL)
-    end
-    vend = :unknown
-    if lib != C_NULL
-        if Libdl.dlsym(lib, :openblas_set_num_threads; throw_error=false) !== nothing
-            vend = :openblas
-        elseif Libdl.dlsym(lib, :openblas_set_num_threads64_; throw_error=false) !== nothing
-            vend = :openblas64
-        elseif Libdl.dlsym(lib, :MKL_Set_Num_Threads; throw_error=false) !== nothing
-            vend = :mkl
-        end
-    end
-    return vend
-end
-end
 
-const _vendor = determine_vendor()
+_vendor = :unknown
+
 vendor() = _vendor
```

Comment:
```suggestion
vendor() = _vendor::Symbol
```

now when it is not const